### ISSUE TITLE
Add option for ASTRAL-Pro? 

### ISSUE THREAD

**Issue by jessiepelosi** [esayyari/DiscoVista#18]
Would it be possible to add an option to use ASTRAL-Pro in place of ASTRAL? Perhaps an flag in the command line to provide a path to ASTRAL-Pro? 

This is a great tool to visualize ASTRAL output, thank you for developing it!

Jessie

**Comment by edgardomortiz** [comment]
Yes, it would be great to be able to use `DiscoVista` on trees with paralogs. Or maybe @smirarab can recommend an alternative method to visualize discordance using this kind of trees?

Edgardo
